src/win32.mak

```diff
@@ -288,10 +288,18 @@ release:
 $G :
 	if not exist "$G" mkdir $G
 
-debdmd:
+check-host-dc:
+	@cmd /c if "$(HOST_DC)" == "" (echo Error: Environment variable HOST_DC is not set & exit 1)
+
+debdmd: check-host-dc debdmd-make
+
+debdmd-make:
+	@echo reldmd
```

Comment:
Why is this line needed?

Reply:
I assume you are asking why have the extra target, why not do this:
```Make
debdmd: check-host-dc 
    $(DMDMAKE) "OPT=" "DEBUG=-D -g -DUNITTEST" "DDEBUG=-debug -g -unittest" "DOPT=" "LFLAGS=-L/ma/co/la" $(TARGETEXE)
```
Instead of this:
```Make
debdmd: check-host-dc debdmd-make

debdmd-make:
    $(DMDMAKE) "OPT=" "DEBUG=-D -g -DUNITTEST" "DDEBUG=-debug -g -unittest" "DOPT=" "LFLAGS=-L/ma/co/la" $(TARGETEXE)
```

The answer is because it doesn't work :)  I don't know why, I'm guess it's a bug in Digital Mars Make, but if you combine them into 1 statement, them make ignores the commands.

Reply:
Oh I was trying to politely ask you to remove the debugging output ;-)
I understand that DigitalMars Make is "rather rudimentary" ...